Makefile

```diff
@@ -8,6 +8,10 @@ BIN_DIR?=bin
 export GOOS?=$(shell go env GOOS)
 export GOARCH?=$(shell go env GOARCH)
 
+depends:
```

Comment:
tried running `make depends` but it errored? `make: [depends] Error 1 (ignored)`

Reply:
I find that worrisome too, visually, but there is a get-out-clause: "ignored" - rest assured that it does not fail (check the return code):
`make depends && echo but I was okay about it`

Reply:
the `grep` fails, but the `-` prefix says "it's okay if grep fails" (i.e. the package isn't found)

Reply:
I (think I 🤔) see, thanks for explaining 👍